src/main/java/appeng/items/parts/PartType.java

```diff
@@ -210,4 +210,19 @@ public GuiText getExtraName()
 		return this.extraName;
 	}
 
+	public Constructor<? extends IPart> getConstructor()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1452)